code/game/objects/items.dm

```diff
@@ -1268,7 +1268,7 @@
 	if((item_flags & ABSTRACT) || HAS_TRAIT(src, TRAIT_NODROP))
 		return
 	user.dropItemToGround(src, silent = TRUE)
-	if(throwforce && HAS_TRAIT(user, TRAIT_PACIFISM))
+	if(throwforce && (HAS_TRAIT(user, TRAIT_PACIFISM)) || HAS_TRAIT(user, TRAIT_NO_THROWING))
```

Comment:
Better solution? I've already gone hogmad with signals and components but this trait feels a bit forced

Reply:
Trait is sensible, though I'll note the way you've currently implemented it they can still bodythrow (which makes the trait itself misleading)

Reply:
I didn't care about bodythrows for voidwalkers implementation but yeah that does make it a bit confusing for others. I'll throw it in